src/app/components/core/system-alert/system-alert.component.ts

```diff
@@ -15,15 +17,23 @@ export class SystemAlertComponent {
   expectedButtonLabels = ['Show Logs', 'Upgrade'];
   sortByKey = 'time';
   showSpinner = true;
+  isManualRefresh = false;
```

Comment:
May be we could use the variable name `isAlive` instead of `isManualRefresh`, which is the standard name for checking ping state throughout the application.

Reply:
We use variable name `isAlive` on the pages wherever we are doing auto refresh call e.g.- South, North etc. `isAlive` indicates that we should keep auto refresh call alive or not.
But here I need to only check that `Auto ping` is set Or `Manual`.
Due to above reason, I think `isManualRefresh` is well suited here.

Reply:
I think Hemant meant that We don't this variable if we have access to pingInfo object, and can use pingInfo.isAlive along with alertsCount